dali/operators/generic/lookup_table.cc

```diff
@@ -51,31 +51,33 @@ void LookupTable<CPUBackend>::RunImpl(SampleWorkspace &ws) {
 DALI_REGISTER_OPERATOR(LookupTable, LookupTable<CPUBackend>, CPU);
 
 DALI_SCHEMA(LookupTable)
-  .DocStr(R"code(Maps input to output by using a lookup table specified by `keys` and `values`
-and a `default_value` for non specified keys.
+  .DocStr(R"code(Maps the input to output by using a lookup table that is specified by
+``keys`` and ``values``, and a ``default_value`` for unspecified keys.
 
-`keys` and `values` are used to define the lookup table::
+For example when ``keys`` and ``values`` are used to define the lookup table in the following way::
 
   keys[] =   {0,     2,   3,   4,   5,    3}
   values[] = {0.2, 0.4, 0.5, 0.6, 0.7, 0.10}
   default_value = 0.99
 
 
-yielding::
+the operator yields::
 
   lut[] = {0.2, 0.99, 0.4, 0.10, 0.6, 0.7}  // only last occurrence of a key is considered
```

Comment:
Mhhh, maybe it should be:
```
{0.2, 0.5, 0.6, 0.7, 0.10, 0.6}
```
@jantonguirao ?

Reply:
no. it's correct. You get the mapping you specified for the keys you specified, and you get the default value (in the example 0.99) for the rest of the keys

Reply:
Also, the key 3 is repeated, so we keep the last occurrence

Reply:
So the input here is  {0, 1, 2, 3, 4, 5}, right? It isn't obvious to me just from reading this, that's that I meant. I

Reply:
Yes, and then you get the values from the `values` that are at the same index as the input value in the `keys`.

Reply:
@awolant Yeah, I agree that the example was confusing because it wasn't said that this is the table that gets populated internally.

The idea is that you populate every element within your minimum and maximum key values. The keys you don't have you fill them with the default value. If you get a key outside of the range you don't even look at the table, you just return the default value